changelog/seqera-cloud/v24.3.0_cycle4.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Seqera Cloud v24.3.0_cycle4
```

Comment:
@swampie, are we naming the release this or adding `cb6efc4`?

Reply:
IMO we should cut it off at cycle, since we increment that with every planned release, right @swampie ?